src/pyodide.js

```diff
@@ -331,31 +331,35 @@ var languagePluginLoader = new Promise((resolve, reject) => {
   Module.preloadedWasm = {};
   let isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
 
-  Module.runPython = function(
-      code) { return Module.py_pyodide.eval_code(code, Module.globals); };
+  // clang-format off
```

Comment:
Do you ever turn this back on?

Reply:
Nope, I forgot about that. I kinda think we should just disable it on the whole file and get a damn js linter.